### PR TITLE
Minor wording changes and corrections

### LINKED ISSUE #15067: Minor wording changes and corrections
Minor suggested changes
- Previous pages used _PMC_ for inline code.  _Powershell_ used here.
- Add migration history table name for additional clarity
- _SqlException_ is referred to as an error.  Suggest referring to it as an exception.

PR will be submitted. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 01013ef6-c55c-ff58-571c-ddc4435993af
* Version Independent ID: 24699fe5-9744-1796-0798-f7b211ec0f2f
* Content: [Add a new field to a Razor Page in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/new-field?view=aspnetcore-3.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/razor-pages/new-field.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/razor-pages/new-field.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by c0g1t8**
Fixes #15067 

Suggestions only :)

### CODE REVIEW

File: aspnetcore/tutorials/razor-pages/new-field.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ Build the solution.
 From the **Tools** menu, select **NuGet Package Manager > Package Manager Console**.
 In the PMC, enter the following commands:
 
-```powershell
+```PMC
```
Comment (scottaddie) [inline aspnetcore/tutorials/razor-pages/new-field.md]:
```suggestion
```powershell
```

File: aspnetcore/tutorials/razor-pages/new-field.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ Another option is to delete the database and use migrations to re-create the dat
 * Select **OK**.
 * In the [PMC](xref:tutorials/razor-pages/new-field#pmc), update the database:
 
-  ```powershell
+  ```PMC
```
Comment (scottaddie) [inline aspnetcore/tutorials/razor-pages/new-field.md]:
```suggestion
  ```powershell
```

**Comment by c0g1t8** [comment]
Thanks for the feedback - this has been a good learning process.

**Comment by c0g1t8** [comment]
I corrected the commit messages.

File: aspnetcore/tutorials/razor-pages/new-field.md [hunk 3]
```diff
@@ -45,13 +45,13 @@ Update the following pages:
 * Update [Create.cshtml](https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie30/Pages/Movies/Create.cshtml) with a `Rating` field.
 * Add the `Rating` field to the Edit Page.
 
-The app won't work until the DB is updated to include the new field. If run now, the app throws a `SqlException`:
+The app won't work until the DB is updated to include the new field. Running the app now would throw a `SqlException`:
 
-`SqlException: Invalid column name 'Rating'.`
+> `SqlException: Invalid column name 'Rating'.`
```
Comment (scottaddie) [inline aspnetcore/tutorials/razor-pages/new-field.md]:
```suggestion
`SqlException: Invalid column name 'Rating'.`
```

File: aspnetcore/tutorials/razor-pages/new-field.md [hunk 4]
```diff
@@ -45,13 +45,13 @@ Update the following pages:
 * Update [Create.cshtml](https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie30/Pages/Movies/Create.cshtml) with a `Rating` field.
 * Add the `Rating` field to the Edit Page.
 
-The app won't work until the DB is updated to include the new field. If run now, the app throws a `SqlException`:
+The app won't work until the DB is updated to include the new field. Running the app now would throw a `SqlException`:
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/razor-pages/new-field.md]: Avoid future tense.

Comment (Rick-Anderson) [inline aspnetcore/tutorials/razor-pages/new-field.md]: The app won't work until the DB is updated to include the new field. Running the app without updating the database throws a `SqlException`:

File: aspnetcore/tutorials/razor-pages/new-field.md [hunk 5]
```diff
@@ -45,13 +45,13 @@ Update the following pages:
...
 
 `SqlException: Invalid column name 'Rating'.`
 
-This error is caused by the updated Movie model class being different than the schema of the Movie table of the database. (There's no `Rating` column in the database table.)
+This exception is caused by the updated Movie model class being different than the schema of the Movie table of the database. (There's no `Rating` column in the database table.)
 
-There are a few approaches to resolving the error:
+There are a few approaches to preventing this exception:
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/razor-pages/new-field.md]: You're really not preventing an exception. It's really an error in what's been done that throws an exception. It's assuming the exception has occurred.

File: aspnetcore/tutorials/razor-pages/new-field.md (hunk 4 above)
Comment (c0g1t8) [reply aspnetcore/tutorials/razor-pages/new-field.md]: Got it - look like I need more time with a style guide :)

Comment (c0g1t8) [reply aspnetcore/tutorials/razor-pages/new-field.md]: I'm assuming this is your preferred wording.  I like it much more.

**Comment by Rick-Anderson** [comment]
@c0g1t8  see my commit

**Pull request merged**